### ISSUE TITLE
CI: run full suite of ansible-test sanity

### ISSUE THREAD

**Issue by robinvy** [ansible-aix/ansible-power-aix#35]
Try running the full suite of ansible-test sanity against our modules to ensure there are no issue for the Galaxy delivery

**Comment by robinvy** [comment]
Found and fixed text formatting issue with pylint:
[https://github.com/ansible-aix/ansible-power-aix/commit/4f609efdd12225c06d8819aef8c3adf6b49ad60b](https://github.com/ansible-aix/ansible-power-aix/commit/4f609efdd12225c06d8819aef8c3adf6b49ad60b)

**All module must include the following lines between header and METEDATA for the ansible-test sanity future-import-boilerplate and metaclass-boilerplate checkings to pass:**
```
from __future__ import absolute_import, division, print_function
__metaclass__ = type
```
fixed lvg as not part of PR#9 (dev-collection)

**Comment by robinvy** [comment]
Adapted travis.yml and devops/bin/sanity_test.sh to run the full sanity test suite automatically.
